src/main/java/com/zaxxer/hikari/pool/PoolBase.java

```diff
@@ -447,7 +448,12 @@ private void checkValidationSupport(final Connection connection) throws SQLExcep
    {
       try {
          if (isUseJdbc4Validation) {
```

Comment:
How does ``isUseJdbc4Validation`` get set to ``true`` if it is not supported? I would have expected ``SQLFeatureNotSupportedException`` to have been thrown earlier, forcing ``isUseJdbc4Validation=false``.

Reply:
@brettwooldridge Good point, I've revised it.